Facebook.Android/nuget/Xamarin.Facebook.Android.nuspec

```diff
@@ -16,6 +16,7 @@
       <group targetFramework="MonoAndroid403">
```

Comment:
I suppose we could add the implied dependencies explicitly...   In this case, AppCompat-v7 pulls in the Vector and Animated Vector drawable packages... the nuspec doesn't explicitly list these.